<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Grupo Nº 1</author>
 <class>GenRndValsDialog</class>
 <widget class="QDialog" name="GenRndValsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>203</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>323</width>
    <height>203</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>323</width>
    <height>203</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generar valores aleatorios</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Generar...</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1" colspan="2">
           <widget class="QCheckBox" name="chkDimensionAleatoria">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>20</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dimensión aleatoria</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="optGenerarEstados">
            <property name="text">
             <string>Estados aleatorios de diferente tipo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QCheckBox" name="chkTecnicaAleatoria">
            <property name="text">
             <string>Técnica aleatoria</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="optGenerarValoresParam">
            <property name="text">
             <string>Valores de los parámetros</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="optGenerarTodo">
          <property name="text">
           <string>Ambos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenRndVals">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Generar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelGenRndVals">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>optGenerarEstados</tabstop>
  <tabstop>chkDimensionAleatoria</tabstop>
  <tabstop>optGenerarValoresParam</tabstop>
  <tabstop>chkTecnicaAleatoria</tabstop>
  <tabstop>optGenerarTodo</tabstop>
  <tabstop>btnGenRndVals</tabstop>
  <tabstop>btnCancelGenRndVals</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnGenRndVals</sender>
   <signal>clicked()</signal>
   <receiver>GenRndValsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelGenRndVals</sender>
   <signal>clicked()</signal>
   <receiver>GenRndValsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
